src/mpi4py/futures/pool.py

```diff
@@ -116,7 +116,7 @@ def submit(self, fn, *args, **kwargs):
             task = (fn, args, kwargs)
             self._pool.push((future, task))
             return future
-    if sys.version_info >= (3, 8):
+    if sys.version_info >= (3, 8):  # pragma: no branch
```

Comment:
What does this pragma do?

Reply:
It is an annotation for test coverage. I usually run it with latest Python, so the false branch is never taken, then coverage marks a miss. https://app.codecov.io/gh/mpi4py/mpi4py

Reply:
Neat!